apps/mobile-wallet/src/components/NFTsGrid.tsx

```diff
@@ -91,7 +90,6 @@ const NFTsGrid = forwardRef(
 export default NFTsGrid
 
 const NFTThumbnailContainer = styled.View`
-  //margin-bottom: ${gap}px;
```

Comment:
not related to this PR, but I was gonna test the NFTs and I see that none is loading. This is our testing wallet which has NFTs on mainnet:



<img width="854" alt="image" src="https://github.com/user-attachments/assets/016c5792-7bd2-48c4-80b4-020c2847b4df" />



<img width="1082" alt="image" src="https://github.com/user-attachments/assets/a0c28b84-ba4e-441a-a752-742c19a66466" />